hack/arktos-dev-env-setup.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash

```

Comment:
Please do not use your personal dev env script here. Many devs have their own preference. Only add necessary script in setup-dev-env.md

Reply:
Removed